assets/src/application/ui/layout/entityList/RegistrationsLink/style.scss

```diff
@@ -0,0 +1,56 @@
+.ee-editor-details-reg-url-link {
+	align-items: center;
+	border: 1px solid transparent;
+	border-radius: var(--ee-border-radius-small);
+	box-shadow: none;
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
+	color: var(--ee-default-text-color);
+	display: inline-flex;
+	font-size: var(--ee-font-size-default);
+	height: var(--ee-icon-button-size);
+	justify-content: center;
+	line-height: var(--ee-font-size-default);
+	margin: 0 auto calc(var(--ee-font-size-bigger) / -4);
+	outline-style: none;
+	text-decoration: none !important;
+	width: var(--ee-icon-button-size);
```

Comment:
some parts of this should go away from here when doing https://github.com/eventespresso/event-espresso-core/issues/2725